sycl/test/group-algorithm/any_of.cpp

```diff
@@ -3,6 +3,7 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
+// XFAIL:cuda
```

Comment:
Could you add a reason for each XFAIL line, please?
According to my understanding we should have 0 expected to fail tests, so I expect to see another TODO comment with instructions how to fix this XFAIL.